docs/source/api_implementation.rst

```diff
@@ -0,0 +1,14 @@
+Which API does it support?
+--------------------------
+========================== ================================================================================================= =====================
+API name                   url                                                                                               Implementation status
```

Comment:
We can save the URL column and introduce the Date Checked.

Specifically, instead of

```
User provisioning API  https://docs.nextcloud.com/server/14/admin_manual/configuration_user/user_provisioning_api.html  OK
```

you can simply write

```
`User provisioning API`_  OK
```

and then just drop a line below the table:

```
.. _User provisioning API: https://docs.nextcloud.com/server/14/admin_manual/configuration_user/user_provisioning_api.html
```

Reply:
Thanks! Added Last checked date column and fixed links